compiler/modules/front/codeGen.bal

```diff
@@ -269,14 +269,14 @@ function codeGenFunction(ModuleSymbols mod, s:FunctionDefn defn, bir:FunctionSig
     }
     var { block: endBlock } = check codeGenStmts(cx, startBlock, { bindings }, defn.body);
     if endBlock != () {
-        bir:RetInsn ret = { operand: () };
+        bir:RetInsn ret = { operand: (), pos: defn.endPos };
         endBlock.insns.push(ret);
     }
-    codeGenOnPanic(cx);
+    codeGenOnPanic(cx, defn.startPos);
```

Comment:
I think the endPos would be better here.

Reply:
fixed